js/apps/system/_admin/aardvark/APP/react/src/views/g6graphs/EdgeStyleSelector.js

```diff
@@ -0,0 +1,73 @@
+import React, { useContext, useState } from 'react';
+import { Select, Tooltip } from 'antd';
+import { InfoCircleFilled } from '@ant-design/icons';
+import { UrlParametersContext } from "./url-parameters-context";
+
+const EdgeStyleSelector = ({ onEdgeStyleChange} ) => {
+  const [urlParameters, setUrlParameters] = useContext(UrlParametersContext);
+  const [type, setType] = useState(urlParameters.edgeType);
+
+  const newUrlParameters = { ...urlParameters };
+
+  const SelectOption = Select.Option;
+
+  const styles = [
```

Comment:
since this is created in render, it will cause multiple re-renders. We can avoid that by moving it out of the component (defined at the top of the file)

Reply:
This is solved as I refactored this component and no antd elements are included anymore (will be included in [FE-196](https://github.com/arangodb/arangodb/pull/18199)). This file was removed from this PR.